great_tables/_utils_render_html.py

```diff
@@ -154,8 +162,8 @@ def create_columns_component_h(data: GTData) -> str:
                 )
             )
 
-        # Join the <th> cells into a string and separate each with a newline
-        th_cells = "\n".join([str(tag) for tag in table_col_headings])
+        # Join the <th> cells into a string and begin each with a newline
```

Comment:
An opportunity was taken here to beautify the HTML output of the column labels portion since snapshots had to be regenerated anyway.